### PR TITLE
CRITICAL FIX: fixes for task builder

### PR DESCRIPTION
**Pull request by travisreed-wf**

### CODE REVIEW

**Comment by travisreed-wf** [comment]
@aguibert your last pull request broke the task builder

**Comment by travisreed-wf** [comment]
2 bugs: 
1 you were checking for the content of the title, not the file upload, 
2 you weren't doing a none check where one was needed

File: src/templates/tasks/tasks_js.js [hunk 1]
```diff
@@ -198,7 +199,8 @@
   };
 
   function upload(f, onsuccess){
-    if($(f).find('input').val().trim().length < 5){
+    if($(f).find('input:file').val().trim().length < 5){
+      console.log("Nothing selected");
```
Comment (AJH693) [inline src/templates/tasks/tasks_js.js]: Log.

**Comment by AJH693** [comment]
otherwise looks good +1

File: src/templates/tasks/tasks_js.js (hunk 1 above)
Comment (travisreed-wf) [inline src/templates/tasks/tasks_js.js]: I want this in there to leave indication to us

Comment (AJH693) [inline src/templates/tasks/tasks_js.js]: Ok

**Pull request merged**